### PR TITLE
Fix issue where expanding an NFT would sometimes produce a duplicate

### PR DESCRIPTION
**Pull request by jxom**
Fixes RNBW-3748

## What changed (plus any additional context for devs)

It seems that when expanding an NFT, it would sometimes produce a duplicate.

Turns out this was an unexpected side-effect when revalidating a collectible. Revalidating collectibles should only happen for ENS NFTs only, and not all NFTs.

The flag to enable revalidation was not guarding against non-ENS NFTs, so I have fixed the `isUnknownOpenSeaENS` function to filter out non-ENS NFTs.

### CODE REVIEW

File: src/parsers/uniqueTokens.js [hunk 1]
```diff
@@ -262,7 +262,7 @@ export const applyENSMetadataFallbackToTokens = async data => {
   return await Promise.all(
     data.map(async token => {
       try {
-        return applyENSMetadataFallbackToToken(token);
+        return await applyENSMetadataFallbackToToken(token);
```
Comment (jxom) [inline src/parsers/uniqueTokens.js:265]: we don't want to swallow errors here

File: src/handlers/ens.ts [hunk 2]
```diff
@@ -122,11 +122,19 @@ const buildEnsToken = ({
...
 };
 
-export const isUnknownOpenSeaENS = (asset?: any) =>
-  asset?.description?.includes('This is an unknown ENS name with the hash') ||
-  !asset?.uniqueId?.includes('.eth') ||
-  !asset?.image_url ||
-  false;
+export const isUnknownOpenSeaENS = (asset?: any) => {
+  const isENS =
+    asset?.asset_contract?.address.toLowerCase() ===
```
Comment (skylarbarrera) [inline src/handlers/ens.ts:127]: wonder if we can use `const isENS = uniqueTokenType === UniqueTokenType.ENS;`

**Review by skylarbarrera — APPROVED** [review]

**Review by estebanmino — COMMENTED** [review]
this was a PR merged to profiles right?

could you rebase to include https://github.com/rainbow-me/rainbow/pull/3587 i can't repro since the app's images aren't working fine

**Comment by jxom** [comment]
@estebanmino – yes it was. I'm pretty sure #3587 is included here already?

**Comment by jxom** [comment]
@estebanmino – Yep it was.

I just rebased!

**Review by ibrahimtaveras00 — APPROVED** [review]
was able to see the fix work with dom.eth FND collection and moxey.eth creature world on both OS's, and sanity checked a few other NFTs and POAPs

QA Passed 👍🏽

**Pull request merged**